src/app/tests/suites/certification/Test_TC_BI_1_1.yaml

```diff
@@ -54,21 +54,12 @@ tests:
 
     #issue #11053 disabled steps below Global attributes missing from YAML framework
```

Comment:
Shouldn't this move down to the EventList bit?

Reply:
I will update this